### DISCUSSION TITLE
ERROR:  usage: pytest [options] [file_or_dir] [file_or_dir] [...]

### DISCUSSION THREAD

**Discussion by kalebers** [pytest-dev/pytest#12422] [Q&A] [answered]
Hello guys,

I'm trying to start my tests running the command `pytest [options] path/to/the/file.py`, but for some reason I'm getting the following error:

`ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --html=./tests/unit/report/unit_test_results.html
  inifile: .....\setup.cfg
  rootdir: ....\....`

Pytest version: pytest 8.2.1
Python version: 3.10
SO: Windows 10

How can I pass this error and run my tests?

**Comment by The-Compiler** [comment, marked as answer]
The relevant part is this:

```
error: unrecognized arguments: --html=./tests/unit/report/unit_test_results.html
```

You're passing a `--html` option, but pytest does not have such an option.

Probably you're missing a plugin that implements it, most likely [pytest-html](https://github.com/pytest-dev/pytest-html).

**Reply by kalebers** [reply]
Thanks, tested with your suggestion and worked :)
